modules/oe_theme_helper/src/Plugin/Block/PageHeaderBlock.php

```diff
@@ -99,4 +124,33 @@ public function setTitle($title): self {
     return $this;
   }
 
+  /**
+   * Returns a list of breadcrumb items ready to be rendered.
+   *
+   * @param string $title
+   *   The title of the current page.
+   *
+   * @return array
+   *   A list of breadcrumb items.
+   */
+  protected function getBreadcrumbSegments($title) {
```

Comment:
You are missing the return value type hint. I see why $title cannot be type hinted.